src/metabase/driver/sync.clj

```diff
@@ -0,0 +1,51 @@
+(ns metabase.driver.sync
+  "General functions and utilities for sync operations across multiple drivers."
+  (:require [clojure.string :as str])
+  (:import java.util.regex.Pattern))
+
+(defn- schema-pattern->re-pattern ^Pattern [schema-pattern]
+  (re-pattern (-> (str/replace schema-pattern #"(^|[^\\\\])\*" "$1.*")
```

Comment:
this could use a comment to explain the regex.

Reply:
Added in next push.